File: Data Structures & Algorithms/two-integer-sum/submission-2.go
func twoSum(nums []int, target int) []int {
    notebook := make(map[int]int)

    for i,value := range nums{
        complement := target - value
        if index,ok := notebook[complement]; ok{
            return []int{index,i} 
        }
        notebook[value] = i
    }
    return []int{}
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array-ii/submission-2.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        swap = 2
        if len(nums) <= 2:
            return 0

        for i in range(2, len(nums)):
            if nums[i] != nums[swap - 2]:
                nums[swap] = nums[i]
                swap += 1
        

        return swap